src/ompl/tools/thunder/SPARSdb.h

```diff
@@ -433,11 +433,21 @@ namespace ompl
                 heuristicScaling_ = heuristicScaling;
             }
 
-            void setUseCostInRoadmap_(const bool useCostInRoadmap)
+            void setUseCostInRoadmap(const bool useCostInRoadmap)
             {
                 useCostInRoadmap_ = useCostInRoadmap;
             }
 
+            void setPathSamplingFactor(const std::optional<int> pathSamplingFactor)
+            {
+                pathSamplingFactor_ = pathSamplingFactor;
+            }
+
+            void setPathSamplingFactor(const int pathSamplingFactor)
+            {
+                pathSamplingFactor_ = pathSamplingFactor;
+            }
```

Comment:
do you need both of these methods? i think you could just keep this one, is there any time you'd call this with out an int argument?

Reply:
good point... I think this one is the only one that we will call